tables/automl/automl_tables_model.py

```diff
@@ -156,12 +167,24 @@ def get_model(project_id, compute_region, model_display_name):
     else:
         deployment_state = "undeployed"
 
+    # get features of top importance
+    feat_list = [
```

Comment:
This looks like it's actually generating an array of tuples, maybe update the comment above. Does the for loop have to come after the tuple syntax?